content/docs/csidriver/installation/operator/_index.md

```diff
@@ -59,9 +59,9 @@ If you have installed an old version of the `dell-csi-operator` which was availa
 | CSI PowerScale     | 1.6       | v6             | 1.19, 1.20, 1.21     | 4.6, 4.7              |
 | CSI PowerScale     | 2.0.0     | v2.0.0         | 1.20, 1.21, 1.22     | 4.6 EUS, 4.7, 4.8     |
 | CSI PowerScale     | 2.1.0     | v2.1.0         | 1.20, 1.21, 1.22     | 4.8, 4.8 EUS, 4.9     |
-| CSI Unity          | 1.6       | v5             | 1.19, 1.20, 1.21     | 4.6, 4.7              |
 | CSI Unity          | 2.0.0     | v2.0.0         | 1.20, 1.21, 1.22     | 4.6 EUS, 4.7, 4.8     |
 | CSI Unity          | 2.1.0     | v2.1.0         | 1.20, 1.21, 1.22     | 4.8, 4.8 EUS, 4.9     |
+| CSI Unity          | 2.2.0     | v2.2.0         | 1.12, 1.22, 1.23     | 4.8, 4.9              |
```

Comment:
@karthikk92 update k8s version from 1.12 to 1.21

Reply:
please update the correct version here.

Reply:
Taken care